Documentation/FAQ/Index.rst

```diff
@@ -41,6 +41,16 @@ Did you configure your solr connection as required?
 
 |
 
+**In which cases do I want to trigger indexing manually?**
+
+- after changing any configuration file.
+- after modifying synonyms, stop words, protected words in TYPO3 Backend -> Search
+
+Moreover by changing core/index configuration you need to reload the core to make the changes become active.
+To reload configuration you can either restart the whole Solr server or simply reload a specific core.
```

Comment:
I believe we added automatic core reload after editing synonyms, stop words, protected words. You might want to double check though. You can also point to the button in the backend module.